duva-client/src/controller.rs

```diff
@@ -89,6 +89,11 @@ impl<T> ClientController<T> {
                 QueryIO::Err(value) => Response::Error(value),
                 _ => Response::FormatError,
             },
+            Append { .. } => match query_io {
```

Comment:
return should be

duva-cli> set 1 2
OK
duva-cli> append 1 2
(integer) 2
duva-cli> append 1 2
(integer) 3
duva-cli> get 1
"222"

Reply:
@Migorithm 
I have changed return value with value length. :)